app/models/ontology_version/proving.rb

```diff
@@ -0,0 +1,38 @@
+module OntologyVersion::Proving
```

Comment:
Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.